src/theme/buttons.ts

```diff
@@ -21,6 +21,16 @@ const buttons: Buttons = {
     },
     outline: {
       primary: {
+        color: colors.blueberryClassic,
+        bgColor: colors.graphite5H,
```

Comment:
According to Figma "no background color in default state" so this should be 'transparent'